src/components/common/WalletList.js

```diff
@@ -218,10 +251,25 @@ class WalletListComponent extends React.PureComponent<Props> {
     }
   }
 
+  listKeyExtractor = (item: WalletListItem, index) => `${item.id}${index}`
   render() {
     const { activeWalletIds, footer, header, wallets } = this.props
     const walletList = this.getWalletList(activeWalletIds, wallets)
-    return <FlatList style={StyleSheet.absoltueFill} data={walletList} renderItem={this.renderRow} ListHeaderComponent={header} ListFooterComponent={footer} />
+
+    return (
+      <SwipeListView
+        keyExtractor={this.listKeyExtractor}
+        data={walletList}
+        style={StyleSheet.absoltueFill}
```

Comment:
misspell

Reply:
what word?

Reply:
absoltueFill

Reply:
Noted, good catch did copy it from the old WalletListScene and yeah need also to change the SortableListView